api/lambdas/sqs-to-converter/index.js

```diff
@@ -306,8 +298,8 @@ export const handler = Sentry.AWSLambda.wrapHandler(async event => {
         } else {
           addExtensionToConversion(conversionResult, documentExtension);
           removePatientFromConversion(conversionResult);
-          addMissingRequests(conversionResult);
         }
+        addMissingRequests(conversionResult);
```

Comment:
we're already looping through thousands of resources - let's add the request in the existing loop instead of making the refactor this way

I would suggest just changing the existing code to the following to minimize further issues:
```
if (!bundleEntry.request) {
          bundleEntry.request = {
```

Reply:
if we were using the existing code we wouldn't have gone through this one.

I feel like this is early optimization, I'd rather minimize code and likelyhood of bugs now than worrying about a few milliseconds of optimization